frontend/README.md

```diff
@@ -122,6 +122,22 @@ In case you need to manually deploy the application, you can use the following c
 bundle exec cap [staging|production] deploy
 ```
 
+## Layers renderization test
+
+In order to test the renderization of the layers, you can use the following command:
+
+```
+yarn test:layers
+```
+
+It only will work if you have the `RUN_LAYERS_TESTS=1` environment variable set in the `cypress.env.json` file. Set to `0` to disable.
```

Comment:
No, I don't think it's a good idea. Because I think you could add your changes accidentally to the repo and it will crash the server due to it is a heavy operation.